jupyter/nbgrader/unit-tests/runUnitTests.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+NBGRADER_LEARNER="learner"
+ASSIGNMENT_NAME="unit-tests"
+
+if [[ "$#" -ne 1 ]]; then
+    echo "Please enter exactly one argument"
+    exit
+fi
+
+NOTEBOOK_FILENAME="$1_tests.ipynb"
+DECODED_FEEDBACK="$1_tests.html"
+
+cd unit-test-env
+
+# # Get kernel language from the submitted assignment's metadata
+kernel_language="$(sed -n 's/"language": //p' "submitted/$NBGRADER_LEARNER/$ASSIGNMENT_NAME/$NOTEBOOK_FILENAME")"
```

Comment:
I'll look into using this, but since this line is also used in one of the scripts in the autograder process, the change would likely only be in the code base and not in the images themselves (not worth going through the entire image update process)